.github/workflows/test.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [v1.25, v1.21]
+        kubernetes: [1.27, 1.24]
```

Comment:
:thinking: I'm not sure, but do we need a `v` here? i.e.
```suggestion
        kubernetes: [v1.27, v1.24]
```

Reply:
Thanks for catching that! The v is needed.